.circleci/config.yml

```diff
@@ -1,14 +1,18 @@
-version: 2
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.2.0
 
 jobs:
-  e2e-test:
+  e2e-test-linux:
```

Comment:
```suggestion
  e2e-test-ubuntu:
```

Reply:
@kotharironak , circleci calls it `Linux Machine` so I tried to follow similar naming scheme here.

Reply:
I'd rather keep it as linux because of what @JBAhire mentions.